internal/pkg/manifest/validate_test.go

```diff
@@ -869,17 +871,18 @@ func TestWorkerService_validate(t *testing.T) {
 					ImageConfig: testImageConfig,
 					TaskConfig: TaskConfig{
 						Platform: PlatformArgsOrString{PlatformString: (*PlatformString)(aws.String("windows/amd64"))},
-						Storage: Storage{Volumes: map[string]*Volume{
-							"foo": {
-								EFS: EFSConfigOrBool{
-									Enabled: aws.Bool(true),
-								},
-								MountPointOpts: MountPointOpts{
-									ContainerPath: aws.String("mockPath"),
+						Storage: Storage{ReadonlyRootFS: aws.Bool(false),
```

Comment:
We don't need to set the `ReadonlyRootFS` field in these test cases (removing the field, gives it a default value of `nil` which should be equivalent to `aws.Bool(false)`). So I think it's safe to remove them from here.